app/grandchallenge/algorithms/templates/algorithms/job_list.html

```diff
@@ -94,11 +112,27 @@ <h5 class="modal-title"></h5>
 
     <script type="text/javascript">
         $('#resultInfoModal').on('show.bs.modal', function (event) {
+            let button = $(event.relatedTarget);
+            let modal = $(this);
+            const json_element = document.getElementById(button.data('pk'));
+            if (json_element){
+                modal.find('.modal-job-output').html(JSON.parse(json_element.textContent));
+            }else {
```

Comment:
I think this means that we can remove the if statement on L28 of jobs_list.html?

```
   {% if job.status == job.SUCCESS %}
        {% for output in job.outputs.all %}{% if output.value %}{{ output.value }}{% endif %}{% empty %}No output produced.{% endfor %}
   {% elif change_job %}
```

would become:

```
   {% if change_job %}
```

Reply:
Well, not completely, if there's no template, the json_element won't be added. The raw result.json is the fallback option.

Reply:
OK!